xbmc/filesystem/HTTPDirectory.cpp

```diff
@@ -105,6 +105,17 @@ bool CHTTPDirectory::GetDirectory(const CURL& url, CFileItemList &items)
         strLinkOptions = strLinkBase.substr(pos);
         strLinkBase.erase(pos);
       }
+      
+      //convert to URL encode
```

Comment:
Please extend comment here to clarify why we're encoding + and ;. Perhaps also add that this list may be extended with other characters.

Reply:
Ok i'm going to add more information thanks